tarpc/src/server.rs

```diff
@@ -77,6 +100,16 @@ pub trait Serve {
 
     /// Responds to a single request.
     async fn serve(self, ctx: context::Context, req: Self::Req) -> Result<Self::Resp, ServerError>;
+
+    /// Wrap this `Serve` in a type that implements [`Stub`].
+    async fn into_stub(self) -> ServeStub<Self>
```

Comment:
Can you remind me why the blanket impl for Serve impls had to be removed?

Reply:
I think the compiler was complaining about conflicting trait implementations.